homeassistant/components/twinkly/light.py

```diff
@@ -136,7 +148,7 @@ def extra_state_attributes(self) -> dict:
 
         # Make sure to update any normalized property
         attributes[ATTR_HOST] = self._client.host
-        attributes[ATTR_BRIGHTNESS] = self._brightness
+        attributes[ATTR_BRIGHTNESS] = self._attr_brightness
```

Comment:
We should just remove the extra state attributes here. Brightness is set by the parent light class in the state attributes, and we don't want static info like host in the state attributes.